specification/attributes/amortization_handling.md

```diff
@@ -0,0 +1,34 @@
+# Amortization Handling
+
+Amortization is a process used to break down and spread purchase costs over the period of time or term of use. 
+
+Amortization is primarily used in purchasable commitment-based discounts. When a purchase is applicable to resources, like commitment-based discounts, the amortized cost of a resource takes the initial payment and term into account and distributes it out based on resource usage, attributing the prorated cost for each hour of billing. Amortization provides a method for FinOps practitioners to reallocate significant purchase charges to the appropriate audience in support of their cost allocation efforts. 
```

Comment:
>  to reallocate significant purchase charges to the appropriate audience

Why "significant", and for example not "full"?
Does the "appropriate audience" refers the resource usages?

To me, the main benefit of amortization is to allocate a purchase cost transparently through the resource usages where it applies.
Sorry, I am not English native speaker: maybe this is what is actually stated here.

Reply:
I just removed 'significant' which should read better.